index.html

```diff
@@ -1893,22 +2013,24 @@ <h3>Profile Signaling</h3>
         <section>
           <h4>Profile Designator</h4>
```

Comment:
nitpick: use plural for the section name: "Profile Designators"

Reply:
Done in f69d320.